src/components/Calculator.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './Calculator.css';
+import PropTypes from 'prop-types';
+
+// Child component
+function Button({ label }) {
+  return <button type="button" className="operators">{label}</button>;
+}
+
+Button.propTypes = {
+  label: PropTypes.string.isRequired,
+};
+
+// Calculator component
+function Calculator() {
+  return (
+    <section className="calc-container">
+      <div className="display">0</div>
+      <div className="keyboard">
+
+        <div className="numbers">
+          <Button label="AC" />
+          <Button label="+/-" />
+          <Button label="%" />
+          <Button label="÷" />
+          <div className="number-buttons">7</div>
+          <div className="number-buttons">8</div>
+          <div className="number-buttons">9</div>
+          <Button label="x" />
+          <div className="number-buttons">4</div>
+          <div className="number-buttons">5</div>
+          <div className="number-buttons">6</div>
+          <Button label="-" />
+          <div className="number-buttons">1</div>
+          <div className="number-buttons">2</div>
+          <div className="number-buttons">3</div>
+          <Button label="+" />
+          <div className="number-buttons-zero-left" />
+          <div className="number-buttons-zero-right">0</div>
+          <div className="number-buttons">.</div>
+          <Button label="=" />
```

Comment:
In the project [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Create%20at%20least%201%20child%20component), it was expected of you to render at least one child component inside here and I think creating a new component to hold the buttons would be a good idea since it will make the code more readable while satisfying the project requirements still.